### DISCUSSION TITLE
Adding math? Subtracting one signal from another

### DISCUSSION THREAD

**Discussion by spongefile** [TelemetryDeck/Viewer#296] [Q&A] [answered]
Hi,

If I have one signal that tells me when something started to load, and another one telling me when something stopped loading, is there a way for me to subtract one from the other and get a graph of average load times per user?

**Comment by winsmith** [comment, marked as answer]
Sadly, this is not possible on a per-signal basis. You could however just send the load time as `floatValue` and then [calculate the average load time](https://telemetrydeck.com/docs/recipes/averagaging-numbers/) in TelemetryDeck.

**Reply by ExecutiveFunctioning** [reply]
Good to know. How do I do a calculation (like calculate average load time) in the TelemetryDeck interface?

**Reply by winsmith** [reply]
Here you go :) https://telemetrydeck.com/docs/recipes/averagaging-numbers/
